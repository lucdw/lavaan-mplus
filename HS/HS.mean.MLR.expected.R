mplus.out <- "HS.mean.MLR.expected.mplus.out" 
lavaan.model <- '
  visual  =~ x1 + x2 + x3
  textual =~ x4 + x5 + x6
  speed   =~ x7 + x8 + x9
'
lavaan.call <-  "sem" 
lavaan.args <- list(
   estimator = "MLR",
   information = "expected",
   meanstructure = TRUE)
test.comment <- ''
if (!exists("group.environment") || is.null(group.environment)) {
   source("../utilities.R", chdir = TRUE)
   execute_test(mplus.out, lavaan.model, lavaan.call, lavaan.args, test.comment)
}
