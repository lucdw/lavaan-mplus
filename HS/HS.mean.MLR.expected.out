Mplus VERSION 8.2
MUTHEN & MUTHEN
01/17/2024   7:42 AM

INPUT INSTRUCTIONS



      title:
      		[This syntax is autogenerated]
      		CFA Mplus syntax
      data:
      		file is HS.raw;
                      listwise=on;
      variable:
      		names are
      		x1  x2  x3  x4  x5  x6
      		x7  x8  x9  ;
      		missing are all (-999999);
      analysis:
      !		model=nomeanstructure;
                      estimator=MLR;
                      information=expected;
                      convergence=0.00000001;
      model:
      		visual  by x1 x2 x3;
      		textual by x4 x5 x6;
      		speed   by x7 x8 x9;
      Output: standardized;









INPUT READING TERMINATED NORMALLY




[This syntax is autogenerated]
CFA Mplus syntax

SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         301

Number of dependent variables                                    9
Number of independent variables                                  0
Number of continuous latent variables                            3

Observed dependent variables

  Continuous
   X1          X2          X3          X4          X5          X6
   X7          X8          X9

Continuous latent variables
   VISUAL      TEXTUAL     SPEED


Estimator                                                      MLR
Information matrix                                        EXPECTED
Maximum number of iterations                                  1000
Convergence criterion                                    0.100D-07
Maximum number of steepest descent iterations                   20

Input data file(s)
  HS.raw

Input data format  FREE



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     X1                    4.936      -0.256       0.667    0.33%       4.000      4.667      5.000
             301.000       1.358       0.330       8.500    0.33%       5.333      5.833
     X2                    6.088       0.472       2.250    0.33%       5.250      5.750      6.000
             301.000       1.382       0.355       9.250    1.66%       6.250      7.000
     X3                    2.250       0.385       0.250    0.33%       1.250      1.750      2.125
             301.000       1.275      -0.894       4.500    1.99%       2.375      3.375
     X4                    3.061       0.269       0.000    0.33%       2.000      2.667      3.000
             301.000       1.351       0.101       6.333    0.33%       3.333      4.000
     X5                    4.341      -0.352       1.000    0.66%       3.000      4.000      4.500
             301.000       1.660      -0.536       7.000    0.33%       4.750      5.500
     X6                    2.186       0.862       0.143    0.66%       1.286      1.857      2.000
             301.000       1.196       0.842       6.143    0.33%       2.286      2.857
     X7                    4.186       0.250       1.304    0.33%       3.304      3.783      4.087
             301.000       1.183      -0.289       7.435    0.33%       4.391      5.130
     X8                    5.527       0.528       3.050    0.33%       4.700      5.250      5.500
             301.000       1.022       1.199      10.000    0.33%       5.700      6.300
     X9                    5.374       0.205       2.778    0.33%       4.500      5.083      5.417
             301.000       1.015       0.312       9.250    0.33%       5.611      6.222


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       30

Loglikelihood

          H0 Value                       -3737.745
          H0 Scaling Correction Factor      1.0435
            for MLR
          H1 Value                       -3695.092
          H1 Scaling Correction Factor      1.0425
            for MLR

Information Criteria

          Akaike (AIC)                    7535.490
          Bayesian (BIC)                  7646.703
          Sample-Size Adjusted BIC        7551.560
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit

          Value                             81.921*
          Degrees of Freedom                    24
          P-Value                           0.0000
          Scaling Correction Factor         1.0413
            for MLR

*   The chi-square value for MLM, MLMV, MLR, ULSMV, WLSM and WLSMV cannot be used
    for chi-square difference testing in the regular way.  MLM, MLR and WLSM
    chi-square difference testing is described on the Mplus website.  MLMV, WLSMV,
    and ULSMV difference testing is done using the DIFFTEST option.

RMSEA (Root Mean Square Error Of Approximation)

          Estimate                           0.090
          90 Percent C.I.                    0.069  0.111
          Probability RMSEA <= .05           0.001

CFI/TLI

          CFI                                0.931
          TLI                                0.897

Chi-Square Test of Model Fit for the Baseline Model

          Value                            880.082
          Degrees of Freedom                    36
          P-Value                           0.0000

SRMR (Standardized Root Mean Square Residual)

          Value                              0.060



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 1.000      0.000    999.000    999.000
    X2                 0.554      0.103      5.359      0.000
    X3                 0.729      0.115      6.367      0.000

 TEXTUAL  BY
    X4                 1.000      0.000    999.000    999.000
    X5                 1.113      0.066     16.762      0.000
    X6                 0.926      0.060     15.497      0.000

 SPEED    BY
    X7                 1.000      0.000    999.000    999.000
    X8                 1.180      0.152      7.758      0.000
    X9                 1.082      0.132      8.169      0.000

 TEXTUAL  WITH
    VISUAL             0.408      0.082      4.966      0.000

 SPEED    WITH
    VISUAL             0.262      0.055      4.762      0.000
    TEXTUAL            0.173      0.055      3.139      0.002

 Intercepts
    X1                 4.936      0.067     73.473      0.000
    X2                 6.088      0.068     89.855      0.000
    X3                 2.250      0.065     34.579      0.000
    X4                 3.061      0.067     45.694      0.000
    X5                 4.341      0.074     58.452      0.000
    X6                 2.186      0.063     34.667      0.000
    X7                 4.186      0.063     66.766      0.000
    X8                 5.527      0.058     94.854      0.000
    X9                 5.374      0.058     92.546      0.000

 Variances
    VISUAL             0.809      0.167      4.837      0.000
    TEXTUAL            0.979      0.121      8.109      0.000
    SPEED              0.384      0.083      4.635      0.000

 Residual Variances
    X1                 0.549      0.138      3.968      0.000
    X2                 1.134      0.107     10.554      0.000
    X3                 0.844      0.085      9.985      0.000
    X4                 0.371      0.050      7.423      0.000
    X5                 0.446      0.058      7.688      0.000
    X6                 0.356      0.046      7.700      0.000
    X7                 0.799      0.079     10.168      0.000
    X8                 0.488      0.074      6.567      0.000
    X9                 0.566      0.068      8.332      0.000


STANDARDIZED MODEL RESULTS


STDYX Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 0.772      0.066     11.657      0.000
    X2                 0.424      0.057      7.424      0.000
    X3                 0.581      0.053     10.867      0.000

 TEXTUAL  BY
    X4                 0.852      0.025     34.175      0.000
    X5                 0.855      0.021     39.779      0.000
    X6                 0.838      0.023     37.022      0.000

 SPEED    BY
    X7                 0.570      0.052     10.971      0.000
    X8                 0.723      0.045     15.992      0.000
    X9                 0.665      0.044     15.003      0.000

 TEXTUAL  WITH
    VISUAL             0.459      0.072      6.366      0.000

 SPEED    WITH
    VISUAL             0.471      0.073      6.457      0.000
    TEXTUAL            0.283      0.075      3.784      0.000

 Intercepts
    X1                 4.235      0.204     20.758      0.000
    X2                 5.179      0.218     23.726      0.000
    X3                 1.993      0.066     29.975      0.000
    X4                 2.634      0.114     23.035      0.000
    X5                 3.369      0.145     23.217      0.000
    X6                 1.998      0.084     23.843      0.000
    X7                 3.848      0.145     26.454      0.000
    X8                 5.467      0.270     20.213      0.000
    X9                 5.334      0.233     22.886      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.404      0.102      3.954      0.000
    X2                 0.821      0.048     16.975      0.000
    X3                 0.662      0.062     10.655      0.000
    X4                 0.275      0.042      6.475      0.000
    X5                 0.269      0.037      7.314      0.000
    X6                 0.298      0.038      7.848      0.000
    X7                 0.676      0.059     11.427      0.000
    X8                 0.477      0.065      7.299      0.000
    X9                 0.558      0.059      9.461      0.000


STDY Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 0.772      0.066     11.657      0.000
    X2                 0.424      0.057      7.424      0.000
    X3                 0.581      0.053     10.867      0.000

 TEXTUAL  BY
    X4                 0.852      0.025     34.175      0.000
    X5                 0.855      0.021     39.779      0.000
    X6                 0.838      0.023     37.022      0.000

 SPEED    BY
    X7                 0.570      0.052     10.971      0.000
    X8                 0.723      0.045     15.992      0.000
    X9                 0.665      0.044     15.003      0.000

 TEXTUAL  WITH
    VISUAL             0.459      0.072      6.366      0.000

 SPEED    WITH
    VISUAL             0.471      0.073      6.457      0.000
    TEXTUAL            0.283      0.075      3.784      0.000

 Intercepts
    X1                 4.235      0.204     20.758      0.000
    X2                 5.179      0.218     23.726      0.000
    X3                 1.993      0.066     29.975      0.000
    X4                 2.634      0.114     23.035      0.000
    X5                 3.369      0.145     23.217      0.000
    X6                 1.998      0.084     23.843      0.000
    X7                 3.848      0.145     26.454      0.000
    X8                 5.467      0.270     20.213      0.000
    X9                 5.334      0.233     22.886      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.404      0.102      3.954      0.000
    X2                 0.821      0.048     16.975      0.000
    X3                 0.662      0.062     10.655      0.000
    X4                 0.275      0.042      6.475      0.000
    X5                 0.269      0.037      7.314      0.000
    X6                 0.298      0.038      7.848      0.000
    X7                 0.676      0.059     11.427      0.000
    X8                 0.477      0.065      7.299      0.000
    X9                 0.558      0.059      9.461      0.000


STD Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 0.900      0.093      9.675      0.000
    X2                 0.498      0.075      6.631      0.000
    X3                 0.656      0.068      9.718      0.000

 TEXTUAL  BY
    X4                 0.990      0.061     16.217      0.000
    X5                 1.102      0.055     20.170      0.000
    X6                 0.917      0.058     15.764      0.000

 SPEED    BY
    X7                 0.619      0.067      9.269      0.000
    X8                 0.731      0.065     11.191      0.000
    X9                 0.670      0.059     11.348      0.000

 TEXTUAL  WITH
    VISUAL             0.459      0.072      6.366      0.000

 SPEED    WITH
    VISUAL             0.471      0.073      6.457      0.000
    TEXTUAL            0.283      0.075      3.784      0.000

 Intercepts
    X1                 4.936      0.067     73.473      0.000
    X2                 6.088      0.068     89.855      0.000
    X3                 2.250      0.065     34.579      0.000
    X4                 3.061      0.067     45.694      0.000
    X5                 4.341      0.074     58.452      0.000
    X6                 2.186      0.063     34.667      0.000
    X7                 4.186      0.063     66.766      0.000
    X8                 5.527      0.058     94.854      0.000
    X9                 5.374      0.058     92.546      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.549      0.138      3.968      0.000
    X2                 1.134      0.107     10.554      0.000
    X3                 0.844      0.085      9.985      0.000
    X4                 0.371      0.050      7.423      0.000
    X5                 0.446      0.058      7.688      0.000
    X6                 0.356      0.046      7.700      0.000
    X7                 0.799      0.079     10.168      0.000
    X8                 0.488      0.074      6.567      0.000
    X9                 0.566      0.068      8.332      0.000


R-SQUARE

    Observed                                        Two-Tailed
    Variable        Estimate       S.E.  Est./S.E.    P-Value

    X1                 0.596      0.102      5.829      0.000
    X2                 0.179      0.048      3.712      0.000
    X3                 0.338      0.062      5.433      0.000
    X4                 0.725      0.042     17.088      0.000
    X5                 0.731      0.037     19.890      0.000
    X6                 0.702      0.038     18.511      0.000
    X7                 0.324      0.059      5.486      0.000
    X8                 0.523      0.065      7.996      0.000
    X9                 0.442      0.059      7.502      0.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.498E-02
       (ratio of smallest to largest eigenvalue)


     Beginning Time:  07:42:41
        Ending Time:  07:42:41
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2018 Muthen & Muthen
